draft-fossati-tls-attestation.md

```diff
@@ -1061,6 +1075,22 @@ the TLS server will terminate the exchange.
 
 TBD.
 
+# Privacy Considerations {#priv-cons}
+
+In background-check mode, the Verifier has access to detailed information about the Attester's TCB through Evidence.
+The Verifier also knows the exact time and the party with whom the secure channel establishment is attempted (i.e., the RP).
+The privacy implications are similar to online OCSP {{-ocsp}}.
+Although the Verifier is trusted by the RP, the same may not apply to the Attester.
+Some ways to address this include:
+
+* Redacting privacy-sensitive evidence claims,
+* Using selective disclosure (e.g., SD-JWT {{-sd-jwt}} with EAT {{-rats-eat}}),
+* Co-locating the Verifier role with the RP,
+* Utilizing privacy-preserving attestation schemes (e.g., DAA {{-rats-daa}}), or
+* Utilizing Attesters manufactured with group identities (e.g., {{FIDO-REQS}}).
+
+Note that the equivalent of OCSP "stapling" involves using a passport topology where the Verifier's involvement is unrelated to the TLS session.
```

Comment:
I'm not sure I understand this sentence. Is this because of caching? Even if there's caching of attestation results, the RP (not the Verifier!) can still use attestation-related information to correlate multiple requests from the same client, even if e.g. the client is behind a proxy.

Reply:
> I'm not sure I understand this sentence.

I am just making the analogy between a stapled OCSP response and an attestation result used as a passport.

> the RP (not the Verifier!) can still use attestation-related information to correlate multiple requests from the same client, even if e.g. the client is behind a proxy.

The RP is a "natural" correlator.  I don't think there's anything we can do about that.

Reply:
I think with DAA/EPID the value prop is that (optionally) the Attester can remain anonymous to the RP (per the [Wikipedia](https://en.wikipedia.org/wiki/Direct_Anonymous_Attestation) text, to the Verifier, but I think Verifier and RP are collocated).

Reply:
> Do you have any text you'd like to suggest? If not, I can give it a try.

TBH,, I don't understand it well enough to come up with good text.

Reply:
OK, for now I have added this simple note: [cdd3b86](https://github.com/tls-attestation/draft-tls-attestation/pull/59/commits/cdd3b86f63c242a29dd4fd8323aaf02a4f8d1d7e)

Reply:
@yaronf please check if [cdd3b86](https://github.com/tls-attestation/draft-tls-attestation/pull/59/commits/cdd3b86f63c242a29dd4fd8323aaf02a4f8d1d7e) addresses your concern.